docs/sdk/authentication/session-sigs/auth-methods/social-login.md

```diff
@@ -36,6 +36,20 @@ async function authWithGoogle() {
 
 At the start of the authentication flow, users will be redirected to the social login page hosted by Lit. Once users have successfully signed in, they will be redirected back to your web app.
 
+In case you want to use a custom OAuth project URL instead of the one provided by Lit, you can pass a callback in the `signIn` method and modify the URL as needed.
```

Comment:
```suggestion
In case you want to use a custom OAuth project instead of the one provided by Lit, you can pass a callback in the `signIn` method and modify the URL as needed. Or you may configure your own `redirectUrl` when initializing the provider
```